src/plugin/adapter-registry/c-tracker.ts

```diff
@@ -58,16 +58,45 @@ export class CTracker extends AdapterVariableTracker {
         }
         this.logger.debug('Resolved', variable.name, { start: variableAddress, size: variableSize });
         const address = BigInt(variableAddress);
+        const startAddress = toHexStringWithRadixMarker(address);
+        const isPointer = this.isMaybePointer(variable, startAddress);
         const variableRange: VariableRange = {
             name: variable.name,
-            startAddress: toHexStringWithRadixMarker(address),
+            startAddress,
             endAddress: variableSize === undefined ? undefined : toHexStringWithRadixMarker(address + variableSize),
             value: variable.value,
             type: variable.type,
+            isPointer,
         };
         return variableRange;
     }
 
+    override async getLocals(session: vscode.DebugSession): Promise<VariableRange[]> {
+        this.getAddressSpaces(session);
```

Comment:
Should this be awaited and / or errors handled?

Reply:
In practice it doesn't need to be awaited. If the REPL output comes in too late, the consequence will be that the "Go to value in memory inspector" context menu command is unavailable for the variables currently in view.

To property await the result, we'd need to set up a callback to be called when we've parsed the REPL output messages and decided that the output is most likely done. Not too complicated. The main decision would be how long to wait if the parsing doesn't go as cleanly as hoped and we aren't sure whether the output is really done. A few hundreds of milliseconds, I suppose. We'd avoid a race condition at a potential performance cost.

Do you have a preference?